### DISCUSSION TITLE
"n" for next match isn't working -- reb-next-match not working?

### DISCUSSION THREAD

**Discussion by dandrake** [kickingvegas/casual-re-builder#17] [Q&A]
I can use "p" for the previous match with your transient window open, but "n" for the next match doesn't work.

This seems to be perhaps an re-builder issue, as without the menu, if I do `C-c C-s` for `reb-next-match`, that doesn't work either, even if I use `M-x` to access it. 

So it's definitely an re-builder issue, but I'm just curious if you've seen this or have any insight...

**Comment by kickingvegas** [comment]
@dandrake Thanks for sharing, unfortunately I am not able to reproduce. Perhaps the current window is not *RE-Builder* ?

**Comment by kickingvegas** [comment]
As this is not a Casual RE-Builder issue, moving to discussion to broaden audience.
